dashboard2/pagecache/sourcecache.md

```diff
@@ -0,0 +1,49 @@
+# Origin Snapshots
+
+Origin Snapshots are your first line of defence against bleedthrough: it is a way of storing and reusing content that was received and picked up from the remote server (in other words, the source site). The goal of Origin Snapshots is to create a controlled snapshot of the source content that you can safely use to translate and serve content over the proxy. 
+
+You might want to set up an Origin Snapshot for publishing content and another to be used during translation. The content of the Snapshot being published should be highly translated, and new content should be collected in a different Snapshot to be used in Preview mode and during translation. 
+
+## Benefits of Origin Snapshots
+
+- If you have at least one Origin Snapshot built and enabled, you have essentially decoupled new content from what you are serving over the proxy (you are in control of untranslated content). 
+  
+  **Note:** Origin Snapshots can only be built by Scans. They will never pick up new content unless you expressly run a Content Extraction to update them.
+  
+  _See [Case Study #1](casestudies.md#case-study-1-freeze-site-before-translation-begins) for an in-depth coverage of this scenario._
```

Comment:
The _Markdown_ file does not exist when the documentation is compiled, this is a 404.